### ISSUE TITLE
fetchTickers(['TRX/USDT', 'XRP/USDT']) returns everything!  (Plus date/time fields missing & questionable real time prices)

### ISSUE THREAD

**Issue by drsavant-pub** [ccxt/ccxt#13856]
- OS: win 8.1
- Programming Language version: nodejs  v14.15.4
- CCXT version: 1.87.40

```
let ccxt = require('ccxt')
let bitrue = new ccxt.bitrue({verbose:true})

bitrue.fetchTickers(['TRX/USDT','XRP/USDT']).then( (results)=>{
  console.log(results)
})
```
The code is not producing an error per se, but rather returning every coin pair regardless of what is provided, and with missing date/time fields.  **(I looked into the ccxt/js/bitrue.js (fetchTickers) and it is using kline data without any reference to a time frame.)**

The output is more massive than gitbash will show, so I've included some here and more in the attached file.

As you can see in addition to a [previous issue](https://github.com/ccxt/ccxt/issues/13487) which noted three fields missing, there are actually seven fields undefined.

### The Main Problem For Me:
**I used a conditional statement comparing the markets/pairs returned with what was submitted, so I have a workaround for the results, but without: timestamp and/or date/time fields in the results I cannot begin to verify how accurate these prices actually are in real time?**

```
 'DEP/USDT': {
    symbol: 'DEP/USDT',
    timestamp: undefined,
    datetime: undefined,
    high: 0.0213,
    low: 0.0213,
    bid: 0.0103,
    bidVolume: undefined,
    ask: 0.0197,
    askVolume: undefined,
    vwap: 0.013505100003820873,
    open: undefined,
    close: 0.0142,
    last: 0.0142,
    previousClose: undefined,
    change: undefined,
    percentage: -0.003333,
    average: undefined,
    baseVolume: 695652.42,
    quoteVolume: 9394.8555,
    info: {
      id: '1554632765',
      last: '0.0142',
      lowestAsk: '0.0197',
      highestBid: '0.0103',
      percentChange: '-0.003333',
      baseVolume: '695652.42',
      quoteVolume: '9394.8555',
      isFrozen: '0',
      high24hr: '0.0213',
      low24hr: '0.0213'
    }
  },
  'AUCTION/USDT': {
    symbol: 'AUCTION/USDT',
    timestamp: undefined,
    datetime: undefined,
    high: 4.89,
    low: 4.89,
    bid: 4.51,
    bidVolume: undefined,
    ask: 4.52,
    askVolume: undefined,
    vwap: 4.660373266124024,
    open: undefined,
    close: 4.53,
    last: 4.53,
    previousClose: undefined,
    change: undefined,
    percentage: 0.000089,
    average: undefined,
    baseVolume: 16505.65,
    quoteVolume: 76922.49,
    info: {
      id: '1859448945',
      last: '4.53',
      lowestAsk: '4.52',
      highestBid: '4.51',
      percentChange: '0.000089',
      baseVolume: '16505.65',
      quoteVolume: '76922.49',
      isFrozen: '0',
      high24hr: '4.89',
      low24hr: '4.89'
    }
  },
  'DEXE/USDT': {
    symbol: 'DEXE/USDT',
    timestamp: undefined,
    datetime: undefined,
    high: 2.86,
    low: 2.86,
    bid: 2.68,
    bidVolume: undefined,
    ask: 2.7,
    askVolume: undefined,
    vwap: 2.752912135639858,
    open: undefined,
    close: 2.7,
    last: 2.7,
    previousClose: undefined,
    change: undefined,
    percentage: 0.000188,
    average: undefined,
    baseVolume: 52449.48,
    quoteVolume: 144388.81,
    info: {
      id: '1163305948',
      last: '2.70',
      lowestAsk: '2.70',
      highestBid: '2.68',
      percentChange: '0.000188',
      baseVolume: '52449.48',
      quoteVolume: '144388.81',
      isFrozen: '0',
      high24hr: '2.86',
      low24hr: '2.86'
    }
  },
...
```
Note: See Attached files
[20220614-bitrue.fetchTickers(1).txt](https://github.com/ccxt/ccxt/files/8905891/20220614-bitrue.fetchTickers.1.txt)
[20220614-bitrue.fetchTickers(2).txt](https://github.com/ccxt/ccxt/files/8905892/20220614-bitrue.fetchTickers.2.txt)

**Comment by kroitor** [comment]
> The code is not producing an error per se, but rather returning every coin pair regardless of what is provided, and with missing date/time fields. (I looked into the ccxt/js/bitrue.js (fetchTickers) and it is using kline data without any reference to a time frame.)

I've added a fix for it in CCXT 1.87.45, please, upgrade in ~20 minutes.

----------

> The Main Problem For Me:
> I used a conditional statement comparing the markets/pairs returned with what was submitted, so I have a workaround for the results, but without: timestamp and/or date/time fields in the results I cannot begin to verify how accurate these prices actually are in real time?

As you can see from the `info` substructure that contains the exchange response _"as is, untouched"_, as well as on this page: https://www.bitrue.com/kline-api/public.json?command=returnTicker – the exchange API does not return the timestamp information for that call, so we do not invent it out of thin air. You have to query the system clock on your own:

```JavaScript
let ccxt = require('ccxt')
let bitrue = new ccxt.bitrue({verbose:true})
bitrue.fetchTickers(['TRX/USDT','XRP/USDT']).then( (results)=>{
  console.log(results)
  const timestamp = bitrue.milliseconds()
  console.log('fetched on', bitrue.iso8601(timestamp))
})
```

Let us know if that does not answer your question.

**Comment by drsavant-pub** [comment]
Since Bitrue doesn't provide the timestamp then there's nothing to be done.  Thanks for trying.  

Thanks for the update.  It works great!
